### DISCUSSION TITLE
Limit the PDF to three pages

### DISCUSSION THREAD

**Discussion by riyash** [mpdf/mpdf#1937] [Q&A]
Hello, 
I am using dynamic content to display in pdf. Everything is working fine but I just want to display 3 pages of PDF regardless of content. It is not necessary to fit the full content in 3 pages. Just need to display 3 pages in pdf and other content can be deleted. 
```
$pdf = new \Mpdf\Mpdf();
$stylesheet = file_get_contents(get_stylesheet_directory_uri().'/style.css'); // external css

$pdf->SetHTMLHeader('<div class="pdf-header" style="margin-bottom:10px;background-color:#0095DB;text-align:center;"><img src="'.$image[0].'" style="width:18%;padding-top:10px;padding-bottom:20px;"></div>');
$pdf->AddPage('', // L - landscape, P - portrait 
        '', '', '', '',
        2, // margin_left
        2, // margin right
       30, // margin top
       10, // margin bottom
        2, // margin header
        2); // margin footer

$pdf->WriteHTML($stylesheet,1);
$pdf->WriteHTML($html,2); 

$pdf->TOC_Entry("content",0);

ob_end_clean();
$pdf->Output();

```
This is the code I am using right now. Please respond as soon as possible

**Comment by riyash** [comment]
Any one can please reply atleast yes or not on this? Is this even possible or not?

**Reply by finwe** [reply]
Directly in mPDF not likely. Trim the PDF with external tool such as ghostscript. Or you can try importing the three pages of the PDF to a new document in mPDF and save the result.

**Reply by riyash** [reply]
@finwe 
I don't have to save the PDF. I have to directly display PDF to the browser with 3 pages only.

**Reply by ssieb** [reply]
So generate it internally and then re-import the result to display?

**Reply by riyash** [reply]
Thanks for the prompt response, but it is a lengthy process and time consuming
